docs/metasploit-framework.wiki/Assigning-Labels.md

```diff
@@ -26,19 +50,19 @@ Plugins and scripts, anything that's not otherwise defined.
 
 ### Module
 
-Touches something in /modules
+Touches something in /modules.
 
-### Specs
```

Comment:
Out of curiosity do we still use this tag at all? I've been thinking of writing more specs recently so not sure if this tag would be beneficial or if it's related to old behavior?

Reply:
`specs` isn't a label that exists currently; there's a `tests` label that's occasionally used, but it's not wired into any automated processes though so it's not particularly noteworthy (in my opinion)

Reply:
Makes sense thanks for clarifying 👍